setup.py

```diff
@@ -26,6 +26,12 @@
     'geopandas',
 ]
 
+processing_require = [
+    'rioxarray',
+    'pyproj',
+    'pathlib'
```

Comment:
pathlib is not necessary I guess, it's part of standard library, right? or is this something special?